src/main/kotlin/com/linchpino/core/repository/AccountRepository.kt

```diff
@@ -29,7 +29,7 @@ interface AccountRepository : JpaRepository<Account, Long>{
     JOIN
```

Comment:
This change can be done in another task. Please revert it!
Related Task:
https://trello.com/c/CSRb35jF/72-update-closestmentortimeslots-native-query-so-that-all-constant-values-be-passed-as-argument-instead-of-hardcode-it